corpus/control-flow.txt

```diff
@@ -495,9 +504,9 @@ end
 
 (program
   (case (identifier)
-    (when (pattern (identifier)) (identifier)
-      (when (pattern (identifier)) (identifier)
-        (else (identifier))))))
```

Comment:
This is another tree change: a series of `when` clauses in a case statement will no longer be nested inside each other.